collectors/APICalls/breakpoints.js

```diff
@@ -274,6 +287,27 @@ const breakpoints = [
                 name: 'getContextAttributes',
                 test: 'var c = document.createElement("canvas"); c.getContext("webgl").getContextAttributes()'
             },
+            //Same as above, but with experimental webgl
+            {
+                name: 'getSupportedExtensions(experimental)',
```

Comment:
I think this is useful as a test (to make sure we are also capturing calls to `experimental-webgl`), but I don't think it will work? Crawler will try to create a breakpoint for `CanvasRenderingContext2D.prototype['getSupportedExtensions(experimental)']` and such a method does not exist. (same with methods below)

Reply:
You're right - I think the existing methods should catch the experimental ones anyway